Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.18
```

Comment:
Any reason for using a tag and not a digest?

Reply:
Ill try using `sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1`  (latest 3.18.2)